app/src/main/java/com/sopt/now/compose/feature/main/MainTab.kt

```diff
@@ -0,0 +1,42 @@
+package com.sopt.now.compose.feature.main
+
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Home
+import androidx.compose.material.icons.filled.Person
+import androidx.compose.material.icons.filled.Search
+import androidx.compose.ui.graphics.vector.ImageVector
+import com.sopt.now.compose.feature.home.navigation.HomeRoute
+import com.sopt.now.compose.feature.my.navigation.MyRoute
+import com.sopt.now.compose.feature.search.navigation.SearchRoute
+
+enum class MainTab(
+    val iconResId: ImageVector,
+    internal val contentDescription: String,
+    val route: String,
+) {
+    HOME(
+        iconResId = Icons.Default.Home,
+        contentDescription = "홈",
+        HomeRoute.ROUTE,
+    ),
+    SEARCH(
+        iconResId = Icons.Default.Search,
+        contentDescription = "검색",
+        SearchRoute.ROUTE,
+    ),
+    MY(
+        iconResId = Icons.Default.Person,
+        contentDescription = "마이페이지",
+        MyRoute.ROUTE,
+    );
+
+    companion object {
+        operator fun contains(route: String): Boolean {
+            return entries.map { it.route }.contains(route)
+        }
+
+        fun find(route: String): MainTab? {
+            return entries.find { it.route == route }
+        }
```

Comment:
```suggestion
        fun find(route: String): MainTab {
            return entries.find { it.route == route } ?: throw IllegalArgumentException("에러메시지")
        }
```

이런식으로 하면 타입 안정성도 챙길 수 있을 것 같고 적절한 예외상황에 대한 대응도 될 수 있을것 같습니다.

Reply:
이렇게 수정하니까 MainTab에 없는 모든 스크린에서 throw가 실행되었습니다...!!
그래서 이대로 두는게 더 좋을것 같아 이대로 뒀습니다 ㅎㅎ